.github/workflows/linkcheck.yml

```diff
@@ -0,0 +1,14 @@
+name: Link checker
```

Comment:
```suggestion
name: Markdown link check
```
And name the file markdown-link-check.yaml.

Reply:
@nwt: Ok, changes made.